changelog.d/100.bugfix

```diff
@@ -0,0 +1 @@
+Explicitly list ``packaging`` as a dependency so ``pipx`` installs it in the virtual environment instead of relying on ``setuptools`` to pull it in.
```

Comment:
wait did it lose my comment? :facepalm:

I was just suggesting renaming this to `100.bugfix.rst`; I think it's a good idea in general to make sure these files have the right extension for their formatting. Sorry if you see this twice.